### ISSUE TITLE
[Accessibility]: Disabling automatic jumping to listing and decompiler in the code browser tool

### ISSUE THREAD

**Issue by diamondStar35** [NationalSecurityAgency/ghidra#8618]
Greetings everyone.
I am using Ghidra with screen readers, spicificly NVDA screen reader, by NV access.
I would like to ask if there is a way to disable automatic jumping to the listing and decompiler when focusing on one of the symbles in the symble tree view. It is not convenient for me to automatically jump to the decompiler especially when I use screen readers since I cannot see the function itself or the symble title until I focus it. By default, focusing on any element in the symble tree view goes to the listing and decompiler part.
Are there work arounds or settings for controling this behavior?
Best regards.

**Comment by dragonmacher** [comment]
First, to clarify, the Listing nor the Decompiler should be taking focus when you interact with the Symbol Tree or Table.   Both the Symbol Tree and Table have options to navigate the tool as symbols are selected.  This feature is on by default.   As you select symbols, the Listing and the Decompiler will change what they are showing.    

The Symbol Table has a toolbar action to disable this feature.  The action's description starts with the text `Toggle on means to navigate to the location...`.     With this action toggled off, the Listing and Decompiler will not update as symbols are selected.  

Apparently, the Symbol Tree does not have a similar action.   We can update the Symbol Tree to add an action to disable the navigation.
